client/app/pages/docs/import.vue

```diff
@@ -105,6 +105,25 @@
               </div>
             </div>
           </div>
+          <!-- Comments -->
+          <div v-if="!backendPending" class="mt-10 grid grid-cols-1 gap-x-6 gap-y-8 sm:grid-cols-6">
+            <div class="sm:col-span-4 space-y-4">
+            <label for="comments" class="block text-sm font-medium leading-6 text-gray-900">Comments</label>
```

Comment:
Regarding `for="comments"` is this label for the following `<RpcTextarea>`? Will `<RpcTextarea>` have the correct DOM id "comments"